Documentation: teaching: fix typo

etzl opened the pull request:
Signed-off-by: Erfan Zamani <erfanzamani3445@gmail.com>

etzl commented on Documentation/teaching/lectures/interrupts.rst:
```diff
@@ -842,24 +842,24 @@ low latency requirements and high frequency:
 
       /* PLEASE, avoid to allocate new softirqs, if you need not _really_ high
          frequency threaded job scheduling. For almost all the purposes
-	 tasklets are more than enough. F.e. all serial device BHs et
```
My editor must be changing these, is that a problem?

dbaluta replied:
yes. we should not modify random lines by adding/removing spaces.

etzl replied:
can I make another commit on top? or should I edit my previous commits?

etzl replied:
No, I meant locally. But I see what you mean, I should edit my last commit using `--amend`

etzl commented:
Ok, I've fixed the line indentations. I reverted my last commit and changed the file. But I realized that I could revert to my first commit and prevent the changes from being committed. So let me know

dbaluta commented:
@etzl looks good now. Only concern is a merge commit which shouldn't be there. Can you please remove it and then force push.

![image](https://user-images.githubusercontent.com/62316/154836157-05da362d-edc4-4a6b-afd4-1a7568b6c3aa.png)

dbaluta commented:
@etzl merged. thanks!

etzl commented:
@dbaluta Glad I could help

tavip reviewed (commented):
LGTM, thank you @etzl !
